internal/prep/prep.go

```diff
@@ -68,6 +68,15 @@ func buildKernelArgumentsFromMCOFile(path string) ([]string, error) {
 			args[2*i+1] = string(val)
 		}
 	}
+
+	if mc.Spec.FIPS {
+		args = append(args, "--karg-append", "fips=1")
+
+		// This is needed because /boot is on a separate partition https://access.redhat.com/solutions/137833
+		// TODO: Should we have this regardless of FIPS?
+		args = append(args, "--karg-append", "boot=LABEL=boot")
```

Comment:
I wonder if we should copy some kargs by default. Not sure though how to identify what should be copied and what not

Reply:
Yeah, I feel like this FIPS thing is just a symptom of some bigger thing we're doing wrong. But I'm not familiar enough with ostree/MCO to say what exactly. We should probably consult with someone who knows